### ISSUE TITLE
Proton only works if the client is attached to a terminal

### ISSUE THREAD

**Issue by ferrelas** [ValveSoftware/Proton#4400]
* Steam client version (build number or date): 1603992687
* Distribution (e.g. Ubuntu): Gentoo
* Opted into Steam client beta?: No
* Have you checked for system updates?: Yes

Proton, or possibly the games themselves, crash immediatly after you attempt to run them, however, if you start steam in a terminal

.steam/steam/steam.sh

as opposed to

steam/steam/steam.sh & exit

it works perfectly, not really a huge issue, but it's annoying to have to have an extra terminal laying around for no reason.

**Comment by aeikum** [comment]
That seems an odd way to launch Steam. Proton works fine for me when I launch it from my DE (e.g. XFCE's menu, or its Application Finder), so you don't need a Terminal open. I'm not an expert in this area - what's the difference between what you're doing and what those DE functions do?

**Comment by bonkmaykr** [comment]
I had a similar issue, which was related to pressure-vessel (or so I was told), but it happened regardless of how I ran it. Changing Proton versions helped for me.

What desktop environment/window manager do you use (DWM, KDE, GNOME), and what version of Steam are you using? (flatpak, official .deb/.rpm, AUR, etc.)

As for this:
> but it's annoying to have to have an extra terminal laying around for no reason.

[Guake](http://guake-project.org/) can reduce desktop clutter, if it annoys you that much.

**Comment by ferrelas** [comment]
Well it works, I reinstalled my OS because I ended up with a situation where I was too out of date to update, and the wiped any symlinks or wherever on my PATH.

I use xfce, and I'm not really sure what way I installed it, pretty sure it's not flatpak at least, it might have been the gentoo ebuild, but it's 4 years ago, so I don't remember all that well. The clutter isn't the major issue, it's more that something is broken.
